frontend/src/components/Ngosignup.js

```diff
@@ -107,12 +109,12 @@ function Ngosignup() {
       avbl.push({"category":"books"});
     if(isCloth)
       avbl.push({"category":"clothes"});
-    console.log(avbl);
     const data = 
     { name: item.name, email: item.email, 
       phoneNumber:item.phoneNumber, registrationNumber:item.registrationNumber, 
       location: { address: item.address, city: item.city, pin_code:item.pin_code }, 
-      password: item.password, confirmPassword: item.confirmPassword , availableItems:avbl
+      password: item.password, confirmPassword: item.confirmPassword , availableItems:avbl,
+      description: item.description
     };
     console.log(data);
```

Comment:
use logger instead of console everywhere

Reply:
For this, we need to add loggers in the frontend folder.